CHANGELOG.md

```diff
@@ -32,18 +32,17 @@ Changelog
 [#158]: https://github.com/GIScience/ohsome-api/issues/158
 [#164]: https://github.com/GIScience/ohsome-api/pull/164
 [#166]: https://github.com/GIScience/ohsome-api/issues/166
+[#174]: https://github.com/GIScience/ohsome-api/issues/174
 [#175]: https://github.com/GIScience/ohsome-api/issues/175
 [#184]: https://github.com/GIScience/ohsome-api/pull/184
 [#191]: https://github.com/GIScience/ohsome-api/issues/191
 
-
```

Comment:
In the past we always put in an extra newline before the `##` heading of a new version. I'd say it would be best to use a consistent style throughout the document.